core/templates/core/diaperchange_list.html

```diff
@@ -17,6 +17,10 @@ <h1>
         {% endif %}
     </h1>
     {% include 'babybuddy/filter.html' %}
+    {% include "mobile/base/list.html" with list_item_template="mobile/diaper_list_item.html" object_list=object_list %}
+    <div class="mobile-list">
+        {% for change in object_list %}<div class="row"></div>{% endfor %}
+    </div>
     <div class="table-responsive">
```

Comment:
This can be removed now, right? I think this is causing an empty second list div:

![image](https://github.com/babybuddy/babybuddy/assets/10456740/45f0f0ef-ff4a-43d6-9dc4-5831b5cad8c1)

Once that's resolved this is good to go!